public/locales/en/common.json

```diff
@@ -54,8 +54,9 @@
         "dateSubmitted": "Date submitted: {{date}}",
         "poi_approved": "Your income statement has been approved. Please be on the lookout for an email from SDC with the discount for future registrations.",
         "poi_pending": "Your income statement is under review.",
+        "poi_declined": "You have been declined for low income discounts. Please contact SDC if you think this is a mistake or submit a new income statement.",
         "bgc_approved": "Your criminal record check has been approved.",
-        "bgc_pending": "Your criminal record check is under review.",
```

Comment:
Is this not being used anymore?

Reply:
we have three states now, so this is also being used.

Reply:
Perhaps I'm misunderstanding but it seems with the new changes, it's only `bgc_approved` and `bgc_declined` and `bgc_pending` is removed?

Reply:
holup yea that shouldn't be removed lol. mb